index.js

```diff
@@ -3582,6 +3582,43 @@
     impl: zipWith
   };
 
+  //# intercalate :: Monoid m => TypeRep m -> m -> [m] -> m
```

Comment:
`[m]` should be `Array m`. `Function a b` is the only type for which we use bespoke notation.